src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -31,6 +31,15 @@ public static void BufferSize_GettingSameAsWindowSize()
     }
 
     [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.NetcoreUwp)]
+    public static void WindowWidth_WindowHeight_InvalidSize_net46()
+    {
+        Assert.Throws<IOException>(() => Console.WindowWidth = 0);
+        Assert.Throws<IOException>(() => Console.WindowHeight = 0);
+    }
+
+    [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
Sorry I just realized I read the conditions backwards and this throws ArgumentOutOfRangeException on core and IOException on desktop but I still don't believe the change adds enough value so I would still make it match desktop.